packages/networks/bin/validate.ts

```diff
@@ -13,6 +20,9 @@ import {
   getAllAddresses,
 } from './utils'
 
+const DAO_CHAIN_ID = 1 // mainnet for now
```

Comment:
Since we are moving to using the safe + zodiac module on every chain, do we still need this?

We would, however need to check that the zodiac module receives its instructions from the Timelock contract?

Reply:
Yes but this will currently lead PRs to fail. We can updated once the Base system is in place.